src/segments/cds.go

```diff
@@ -0,0 +1,68 @@
+package segments
+
+import (
+	"encoding/json"
+	"oh-my-posh/environment"
+	"oh-my-posh/properties"
+)
+
+type Cds struct {
+	language
+	HasCDSDependency bool
```

Comment:
As the context is already CDS, does it makes sense to rename this to `HasDependency`?

Reply:
sounds ok